src/Compilers/CSharp/Test/Symbol/Symbols/LocalFunctionTests.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System.Linq;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Symbols
+{
+    public class LocalFunctionTests : CSharpTestBase
+    {
+        [Fact, WorkItem(27719, "https://github.com/dotnet/roslyn/issues/27719")]
+        public void LocalFunctionIsNotStatic()
+        {
+            var source = @"
+class C
+{
+    void M()
+    {
+        void local() {}
+        local();
+    }
+}";
+            var compilation = CreateCompilation(source).VerifyDiagnostics();
+            var tree = compilation.SyntaxTrees[0];
+            var semanticModel = compilation.GetSemanticModel(tree);
+
+            var localSyntax = tree.GetCompilationUnitRoot().DescendantNodes().OfType<LocalFunctionStatementSyntax>().Single();
+            var local = semanticModel.GetDeclaredSymbol(localSyntax);
+            Assert.False(local.IsStatic);
```

Comment:
Please also assert the expected return values of `local.RequiresInstanceReceiver`.